src/components/routerView.browser.spec.ts

```diff
@@ -67,7 +67,7 @@ test('renders components for initial route', async () => {
   expect(app.html()).toBe('Child')
 })
 
-test('updates components when route changes', async () => {
+test.only('updates components when route changes', async () => {
```

Comment:
```suggestion
test('updates components when route changes', async () => {
```
Assuming that was for development